src/App.tsx

```diff
@@ -43,6 +45,21 @@ function App() {
       <div className="App">
         <Header />
         <Switch>
+          <Route path="/terms-and-conditions">
+            <div className={"big-old-container"}>
+              <ReactMarkdown>{terms}</ReactMarkdown>
```

Comment:
Could you replace these 

```
<ReactMarkdown>...</ReactMarkdown>
```

blocks with your new components?

ie

```
<TermsAndConditions />
```

@gpoliko